src/main/kotlin/tanvd/grazi/ide/GraziInspection.kt

```diff
@@ -18,73 +20,87 @@ class GraziInspection : LocalInspectionTool() {
     companion object : GraziLifecycle {
         private fun getProblemMessage(fix: Typo): String {
             if (ApplicationManager.getApplication().isUnitTestMode) return fix.info.rule.id
-
-            val message = if (fix.isSpellingTypo) {
-                //language=HTML
-                """
-                    <html>
-                        <body>
-                            <div>
-                                <p>${fix.info.rule.toDescriptionSanitized()}</p>
-                            </div>
-                        </body>
-                    </html>
-                """.trimIndent()
-            } else {
-                val examples = fix.info.incorrectExample?.let {
-                    val corrections = it.corrections.filter { it?.isNotBlank() ?: false }
-                    if (corrections.isEmpty()) {
-                        //language=HTML
-                        """
-                            <tr style='padding-top: 5px;'>
-                                <td style='color: gray;'>Incorrect:</td>
-                                <td>${it.toIncorrectHtml()}</td>
-                            </tr>
-                        """.trimIndent()
-
-                    } else {
-                        //language=HTML
-                        """
-                            <tr style='padding-top: 5px;'>
-                                <td style='color: gray;'>Incorrect:</td>
-                                <td style='text-align: left'>${it.toIncorrectHtml()}</td>
-                            </tr>
-                            <tr>
-                                <td style='color: gray;'>Correct:</td>
-                                <td style='text-align: left'>${it.toCorrectHtml()}</td>
-                            </tr>
-                        """.trimIndent()
+            return createHTML(false).html {
+                body {
+                    div {
+                        style = "margin-bottom: 5px;"
+                        if (fix.isSpellingTypo) {
+                            div {
+                                if (fix.info.rule.description.length > 50) {
+                                    style = "width: 300px;"
+                                }
+
+                                p { unsafe { +fix.info.rule.toDescriptionSanitized() } }
```

Comment:
Instead of using `unsafe` you can just craate extension on `p` tag :)

Reply:
Fixed